sdk/storage/azure-storage-queue/azure/storage/queue/_shared/parser.py

```diff
@@ -0,0 +1,14 @@
+import sys
+
+if sys.version_info < (3,):
+    def _str(value):
+        if isinstance(value, unicode):  # pylint: disable=undefined-variable
+            return value.encode('utf-8')
+
+        return str(value)
+else:
+    _str = str
+
+
+def _to_utc_datetime(value):
+    return value.strftime('%Y-%m-%dT%H:%M:%SZ')
```

Comment:
should this be deserialization/serialization code instead?

Reply:
It seems this was used in both serialization and deserialization so I named it parser: parse from on object to the other haha